src/DebuggerData.cpp

```diff
@@ -321,18 +332,57 @@ int Breakpoints::findBreakpoint(quint16 addr)
 			   }
    			);
 	if (i != breakpoints.end() && i->address == addr) {
-		return i - breakpoints.begin();
+		currentIndex = i - breakpoints.begin();
+	} else {
+		currentIndex = -1;
 	}
-	return -1;
+	return currentIndex;
 }
 
 int Breakpoints::findNextBreakpoint()
 {
-	// stub
-	// will implement findfirst/findnext scheme for speed
+	if (currentIndex < breakpointCount() - 1) {
+		return ++currentIndex;
+	}
 	return -1;
 }
 
+QStringList AttributeNames = {
+	"type",
+	"id",
+	"address",
+	"regionEnd",
+	"ps",
+	"ss",
+	"segment",
+	"condition",
+};
```

Comment:
This can be `static const`.
Consider moving it inside the getAttribute() method.
(detail) Consider making it a plain array instead of a `QStringList`.
But see below, this list may not be needed at all.

Reply:
properly fixed somewhere else